rasa_core/train.py

```diff
@@ -113,6 +122,12 @@ def train_dialogue_model(domain_file, stories_file, output_path,
                                                  "debug_plots"})
     training_data = agent.load_data(stories_file, **data_load_args)
 
+    if url is not None:
```

Comment:
should go into a separate function (probably this block as well as the line before with the data loading, so that all the data loading is in one function)